voltdbclient/url.go

```diff
@@ -25,9 +26,23 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
-		if u.Port() == "" {
+		if getPort(u.Host) == "" {
 			u.Host = u.Host + ":21212"
 		}
 	}
 	return u, nil
 }
+
+func getPort(host string) string{
```

Comment:
Thank you @mehmetalisavas and @gernest for the clarification.

@mehmetalisavas, I think it's fine to include this function in our source as long as you clearly state where it was originally from for both attribution and future reminder to remove purposes. Once you update this PR with that code comment, I'm fine merging it. Thanks!